### PR TITLE
Automate updating from remote data sources

### PR DESCRIPTION
**Pull request by gsoltis**
Adds a Github Action to update the source data and push to the repo twice a day.

### CODE REVIEW

**Comment by gsoltis** [comment]
I've verified that when set to trigger on push, this action will update my branch with the results of running the scripts. I have not observed the scheduled events, but I am modeling the trigger off of the daily cron from `covid-data-model` that seems to work.

**Comment by gsoltis** [comment]
Here is a sample automated commit (link may 404 eventually, since the commit is no longer referenced) https://github.com/covid-projections/covid-data-public/commit/6ac8960ff720a810a6ce28c8d26cb5aabdb27893

File: push_update.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+now=$(date)
+git config --local user.email "action@github.com"                                            
+git config --local user.name "GitHub Action"                                                 
+git add -A                                                                                   
+git commit -m "Update source data at $now"                                                   
+git push "https://${GITHUB_TOKEN}@github.com/${GITHUB_REPOSITORY}.git" HEAD:${GITHUB_REF}
```
Comment (ghop02) [inline push_update.sh:6]: You could take this one step further and create a PR using the new github cli!
https://github.com/cli/cli

Comment (mikelehen) [reply push_update.sh:6]: Create a PR and then automatically merge it?  I think just pushing directly is probably good.

Comment (ghop02) [reply push_update.sh:6]: haha no automatic merge! Just creating the actual pr.  

Ohh didn't realize that this pushes to master!  do we have any sanity checks on the data? This seems fine for now though

**Comment by gsoltis** [comment]
@mikelehen and @ghop02 My inclination is not to do a PR since I think it's added complexity, but if there's a particular reason to do that vs a single commit, I can look into it. FWIW, it appears that github cli does not yet support merging PRs, but there may be other options: https://github.com/cli/cli/issues/373

**Comment by gsoltis** [comment]
Fixes #24

**Pull request merged**